assets/components/ovn/master/daemonset.yaml

```diff
@@ -120,32 +120,26 @@ spec:
           trap quit TERM INT
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
+          compact() {
+            while true; do
+              /usr/bin/ovn-appctl -t /var/run/ovn/ovn${1}_db.ctl --timeout=5 ovsdb-server/compact 2>/dev/null || true
+              sleep 600
```

Comment:
I was going to suggest inverting sleep and ovn-appctl , to give the process some breathing room at startup, but that would come at the cost of harder testing, now that is triggered right away, and if there is an error we will find it If we exchange the sleep and appctl errors could be hidden.

Reply:
+1

Reply:
Maybe a middle ground: `sleep 15` before while loop? Or other adequate value